website/AboutMe.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Omar Alhazmi | Portfolio </title>
+    <link rel="stylesheet" href="css/styles.css"/> 
+</head>
+<header>
+    <nav>
+    <ul class="nul">
+        <li class="nli"><a class="active" href="/Users/mac/sei/homeworks/hw-w02d01-website/website/AboutMe.html">About Me</a></li>
+        <li class="nli"><a href="/Users/mac/sei/homeworks/hw-w02d01-website/website/Portfolio.html">Portfolio</a></li>
```

Comment:
be careful with file locations, use relative file locations in the future to reduce the number of errors you have in the future.